ripc-core/src/main/java/io/ripc/internal/Publishers.java

```diff
@@ -0,0 +1,48 @@
+package io.ripc.internal;
+
+import org.reactivestreams.Publisher;
+import org.reactivestreams.Subscriber;
+import org.reactivestreams.Subscription;
+
+/**
+ * Temporary utility class for creating and transforming {@link Publisher}s.
+ */
+public class Publishers {
+
+    public static <T> Publisher<T> just(final T value) {
+        return new Publisher<T>() {
+            @Override
+            public void subscribe(final Subscriber<? super T> s) {
+                s.onSubscribe(new Subscription() {
+                    @Override
+                    public void request(long n) {
+                        s.onNext(value);
```

Comment:
There should be a positive demand check here per the spec. The `Specifications` class was intended to be the helper that aggregated these checks. This will be an ongoing issue (proper adherence to the spec) any time we implement helpers in the core.